sway/tree/container.c

```diff
@@ -989,3 +992,82 @@ void container_get_box(struct sway_container *container, struct wlr_box *box) {
 	box->width = container->width;
 	box->height = container->height;
 }
+
+/**
+ * Translate the container's position as well as all children.
+ */
+static void container_floating_translate(struct sway_container *con,
+		double x_amount, double y_amount) {
+	con->x += x_amount;
```

Comment:
We could assert that this is a floating container here

Reply:
I added the assert in the public function because it seemed like a more appropriate place. Also, `container_floating_translate` will get called of children of floating containers once we implement that, so putting the assertion here wouldn't work anyway.

Reply:
Yeah, +1